src/Tickets/Commerce/Gateways/Stripe/Stripe_Elements.php

```diff
@@ -47,7 +48,10 @@ public function get_checkout_template_vars() {
 	 * @return bool
 	 */
 	public function include_payment_element() {
-		return tribe_get_option( Settings::$option_checkout_element ) === Settings::PAYMENT_ELEMENT_SLUG;
+		$payment_methods = ( new Merchant() )->get_payment_method_types();
+		$card_only       = ( 1 === count( $payment_methods ) && 'card' === $payment_methods[0] ) ? true : false;
+		$payment_setting = tribe_get_option( Settings::$option_checkout_element ) === Settings::PAYMENT_ELEMENT_SLUG ? true : false;
+		return $payment_setting && ! $card_only;
```

Comment:
Test `$payment_setting` first to save time - just return false if not true. Spares you having to instantiate the Merchant object. Also once you do, don't bother with the ternary, just return the results with an or: (double-check my logic before applying)
```suggestion
		if ( tribe_get_option( Settings::$option_checkout_element ) !== Settings::PAYMENT_ELEMENT_SLUG  ) {
			return false;
		}
		$payment_methods = ( new Merchant() )->get_payment_method_types();

		return ! 1 !== count( $payment_methods ) || 'card' !== $payment_methods[0];
```